Source/Core/InputCommon/ControllerInterface/DInput/DInputJoystick.cpp

```diff
@@ -46,20 +46,18 @@ void InitJoystick(IDirectInput8* const idi8, HWND hwnd)
                   js_device->SetCooperativeLevel(nullptr, DISCL_BACKGROUND | DISCL_NONEXCLUSIVE)))
           {
             // PanicAlert("SetCooperativeLevel failed!");
-            js_device->Release();
             continue;
           }
         }
 
-        auto js = std::make_shared<Joystick>(js_device);
+        auto js = std::make_shared<Joystick>(js_device.Get());
```

Comment:
This should be passed as a smart pointer if the Joystick is now going to have ownership of it.. otherwise js will be referencing an invalid device?